package.json

```diff
@@ -34,10 +34,10 @@
     "example": "examples"
   },
   "scripts": {
-    "test": "bin/test",
-    "start": "bin/browse",
-    "package": "bin/package",
-    "postinstall": "bin/publish"
+    "test": "bin/snuggsi test",
```

Comment:
@brandondees @rianby64 @tmornini wow can resolve merge conflicts from Github now. :-)

@rianby64 took care of it and merged into your branch as it was my doing from merging #175 earlier today.